example/tests/test_backends.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from rest_framework.test import APITestCase
+
+from ..models import Blog, Entry
+
+ENTRIES = "/nopage-entries"
+
+
+class DJATestParameters(APITestCase):
+    """
+    tests of JSON:API backends
+    """
+    fixtures = ('blogentry',)
+
+    def setUp(self):
+        self.entries = Entry.objects.all()
+        self.blogs = Blog.objects.all()
+
+    def test_sort(self):
+        """
+        test sort
+        """
+        response = self.client.get(ENTRIES + '?sort=headline')
```

Comment:
parameters can be passed on as data e.g. in this case data={'sort': 'headline'}. This way there is no need for string concats